Src/Managers/font_manager/font_manager.py

```
@@ -0,0 +1,90 @@
+
+@singleton
+class FontManager:
+    __logger = lateinit(logging(), 'managers')
+    __registry: str | int
+    fonts: dict[str, dict[int, FontUnit]]
+    default: FontUnit
+
+
+    def __init__(self, path_config: Path):
+        if not path_config.exists():
+            self.__logger.error(f"Не существует конфигационного файла {font_config}")
```

Ruff (0.15.20)

[warning] String contains ambiguous `Н` (CYRILLIC CAPITAL LETTER EN). Did you mean `H` (LATIN CAPITAL LETTER H)?

(RUF001)